LANGS.md

```diff
@@ -1 +1,2 @@
-* [English](en/)
\ No newline at end of file
+* [English](en/)
+* [Français](fr/)
```

Comment:
@MattiSG  Why adding content to the French documentation, and not the English one ?